specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -596,7 +602,7 @@
                 },
                 "restartPolicy": {
                   "type": "string",
-                  "description": "- `always` Always restart\n",
+                  "description": "Restart policy for all containers within the container group. Only option is Always.",
```

Comment:
The enum value is "always," with lowercase 'a' -- should we maintain consistency with that? Also, a slightly modified option here, but might make it sound less like a limit, and perhaps even hint at future options:

"Restart policy for all containers within the container group. The currently available option is `always`."